presto-main/src/main/java/com/facebook/presto/execution/DropColumnTask.java

```diff
@@ -0,0 +1,73 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.execution;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.metadata.Metadata;
+import com.facebook.presto.metadata.QualifiedObjectName;
+import com.facebook.presto.metadata.TableHandle;
+import com.facebook.presto.security.AccessControl;
+import com.facebook.presto.spi.ColumnHandle;
+import com.facebook.presto.sql.analyzer.SemanticException;
+import com.facebook.presto.sql.tree.DropColumn;
+import com.facebook.presto.sql.tree.Expression;
+import com.facebook.presto.transaction.TransactionManager;
+import com.google.common.util.concurrent.ListenableFuture;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.facebook.presto.metadata.MetadataUtil.createQualifiedObjectName;
+import static com.facebook.presto.sql.analyzer.SemanticErrorCode.MISSING_COLUMN;
+import static com.facebook.presto.sql.analyzer.SemanticErrorCode.MISSING_TABLE;
+import static com.facebook.presto.sql.analyzer.SemanticErrorCode.NOT_SUPPORTED;
+import static com.google.common.util.concurrent.Futures.immediateFuture;
+import static java.util.Locale.ENGLISH;
+
+public class DropColumnTask
+        implements DataDefinitionTask<DropColumn>
+{
+    @Override
+    public String getName()
+    {
+        return "DROP COLUMN";
+    }
+
+    @Override
+    public ListenableFuture<?> execute(DropColumn statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
+    {
+        Session session = stateMachine.getSession();
+        QualifiedObjectName tableName = createQualifiedObjectName(session, statement, statement.getTable());
+        Optional<TableHandle> tableHandle = metadata.getTableHandle(session, tableName);
+
+        String column = statement.getColumn().toLowerCase(ENGLISH);
+
+        if (!tableHandle.isPresent()) {
+            throw new SemanticException(MISSING_TABLE, statement, "Table '%s' does not exist", tableName);
+        }
+        accessControl.checkCanRenameColumn(session.getRequiredTransactionId(), session.getIdentity(), tableName);
+
+        Map<String, ColumnHandle> columnHandles = metadata.getColumnHandles(session, tableHandle.get());
+        if (!columnHandles.containsKey(column)) {
+            throw new SemanticException(MISSING_COLUMN, statement, "Column '%s' does not exist", column);
+        }
+        if (columnHandles.size() == 1) {
+            throw new SemanticException(NOT_SUPPORTED, statement, "Cannot drop column '%s'. It is the only column in the table", column);
```

Comment:
End the second sentence with a period.

Rationale: Normally, error messages are just phrases, so they don't end in a period. But if we have multiple sentences, all of them should end in a period.